vsphere/6.7/vcsa/inspec/vmware-vcsa-6.7-stig-baseline/photon/controls/PHTN-67-000009.rb

```diff
@@ -39,15 +41,15 @@
 'SRG-OS-000393-GPOS-00173', 'SRG-OS-000396-GPOS-00176',
 'SRG-OS-000423-GPOS-00187']
   tag gid: 'V-239081'
-  tag rid: 'SV-239081r675051_rule'
+  tag rid: 'SV-239081r816597_rule'
   tag stig_id: 'PHTN-67-000009'
   tag fix_id: 'F-42251r675050_fix'
   tag cci: ['CCI-000068', 'CCI-001453', 'CCI-002418', 'CCI-002450',
 'CCI-002890']
   tag nist: ['AC-17 (2)', 'AC-17 (2)', 'SC-8', 'SC-13', 'MA-4 (6)']
 
-  describe command('sshd -T|&grep -i FipsMode') do
-    its ('stdout.strip') { should cmp 'FipsMode yes' }
+  describe command('sshd -T|&grep -i fipsMode') do
```

Comment:
Was there an issue with the case? Should have been ok as it was.  cmp doesn't care about case and the command with -i is case insensitive

Reply:
No there was not. I just changed the case to match the wording in the STIG text since that was one of their changes.